packages/preview/matset/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "matset"
+version = "0.1.0"
+entrypoint = "api.typ"
+authors = ["@OptimisticPeach", "patrikbuhring@gmail.com"]

```

Comment:
If there is another contributor it would be nice to use the following format:

```toml
authors = ["OptimisticPeach <@OptimisticPeach>", "Their name / username <patrikbuhring@gmail.com>"]
```

If its your own email address, then you should probably do something like:

```toml
authors = ["OptimisticPeach <patrikbuhring@gmail.com>"]
```

Otherwise it will not be parsed and displayed as expected.

Reply:
Nope, I am Patrik Buhring, so I'll use the second suggestion.